.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Documentation
+about: Report an issue in our docs
+title: "[Docs] "
+labels: 'documentation'
+assignees: ''
+
+---
+
+**Description**
+Describe the issue that you've encountered with our documentation.
+
```

Comment:
Should this also include a section that says, like, "suggested fix" or something like that? It'd be ideal if they suggested clearer language (or w/e) instead of just reporting docs problems

Reply:
yeah good call